test.sh

```diff
@@ -16,7 +16,17 @@ curl -sSfg -u $SK: $HOST/v1/customers/$cus \
 curl -sSfg -u $SK: $HOST/v1/customers/$cus \
      -d preferred_locales[]='fr-FR' -d preferred_locales[]='es-ES'
 
+cusB=$(curl -sSfg -u $SK: $HOST/v1/customers \
+            -d email=george.carlin@example.com \
+       | grep -oE 'cus_\w+' | head -n 1)
+
+email=$(curl -sSfg \
+             -u $SK: $HOST/v1/customers?email=james.robinson@example.com \
+        | grep -oE '"email": "james.robinson@example.com"')
```

Comment:
Can you grep the whole line like at other places? (with the comma at the end)

Reply:
OK, so the commas are also missing in the lines 795 800 805 926 935, in the `grep -oE '"total_count":...`

Reply:
That's true, we missed it during previous reviews!
Feel free to open another PR to fix that if you want ;)